identity-account/src/account/account.rs

```diff
@@ -153,13 +154,12 @@ impl Account {
     Ok(snapshot)
   }
 
-  /// Updates the identity specified by the given `key` with the given `command`.
-  pub async fn update_identity<K: IdentityKey>(&self, key: K, command: Command) -> Result<()> {
-    let identity: IdentityId = self.try_resolve_id(key).await?;
-
-    self.process(identity, command, true).await?;
-
-    Ok(())
+  /// Return the `IdentityUpdater` for the given `key`.
+  ///
+  /// On this type, various operations can be executed
+  /// that modify an identity, such as creating services or methods.
+  pub fn update_identity<K: IdentityKey + Clone>(&self, key: K) -> IdentityUpdater<'_, K> {
```

Comment:
Nit: comment suggestion. Ignore if we want to exclude `Command` from the public docs entirely but I think we need some way to reference the allowed operations.

```suggestion
  /// Return an `IdentityUpdater` for the given `key`.
  ///
  /// Allows executing various commands that modify an identity, 
  /// such as adding or removing services and methods.
  ///
  /// See [`Command`] for more details.
  pub fn update_identity<K: IdentityKey + Clone>(&self, key: K) -> IdentityUpdater<'_, K> {
```

Reply:
Agreed. Perhaps we should simply pull the methods that create the builders out of the macro and implement them by hand. We could then document each operation on these methods.

Then we can also exclude the `create_identity` builder method, although it can also be removed by removing the macro invocation for it.

Another approach for the documentation would be to summarize it on the `IdentityUpdater` struct documentation.

Reply:
I concur. Manually implementing the functions for the `IdentityUpdater` would be good since developers currently lack e.g. line completion or even a complete list of builder functions without looking at the docs. Hopefully there's a way to still use macros internally to avoid too much duplicate code though.

Enumerating the list of builder methods available for the `IdentityUpdater` in its doc-comment may be useful to reference but not ideal.